src/Surfnet/StepupMiddleware/ApiBundle/Identity/Projector/RaSecondFactorProjector.php

```diff
@@ -193,6 +194,7 @@ public function applyEmailVerifiedEvent(EmailVerifiedEvent $event)
     public function applySecondFactorVettedEvent(SecondFactorVettedEvent $event)
     {
         $this->updateStatus($event->secondFactorId, SecondFactorStatus::vetted());
+        $this->updateDocumentNumber($event->secondFactorId, $event->documentNumber);
```

Comment:
Even though the entire command handling pipeline is wrapped in a transaction, it would be nice to have the status and document number be updated in one transaction within the context of the projector (ie. inline `updateStatus()` and `updateDocumentNumber()` and call `save()` once). My opinion, feel free to disagree.